cns/service/main.go

```diff
@@ -104,9 +106,11 @@ const (
 	// envVarEnableCNIConflistGeneration enables cni conflist generation if set (value doesn't matter)
 	envVarEnableCNIConflistGeneration = "CNS_ENABLE_CNI_CONFLIST_GENERATION"
 
-	cnsReqTimeout          = 15 * time.Second
-	defaultLocalServerIP   = "localhost"
-	defaultLocalServerPort = "10090"
+	cnsReqTimeout                    = 15 * time.Second
+	defaultLocalServerIP             = "localhost"
+	defaultLocalServerPort           = "10090"
+	defaultDevicePluginRetryInterval = 2 * time.Second
```

Comment:
Can we add some description comments on - Why one is 2 seconds and the other one is 5 seconds?